src/pages/experiments/[experimentId]/data-exploration/components/generic-gene-table/ListSelected.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Button, Select,
+} from 'antd';
+import { useSelector } from 'react-redux';
+
+const ListSelected = () => {
+  const selectedGenes = useSelector((state) => state.genes.selected);
+  const [listed, setListed] = useState(false);
+
+  useEffect(() => {
+    setListed(false);
+  }, [selectedGenes])
```

Comment:
you don't need to have this useEffect at all. Any specific reason why you added it?